modules/serverless-rn-1-30-0.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-30-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.30
+
+{ServerlessProductName} 1.30 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[IMPORTANT]
+====
+{ocp-product-title} 4.13 is based on {op-system-base-full} 9.2.  {op-system-base} 9.2 is yet to be submitted for Federal Information Processing Standards (FIPS) validation. Although Red Hat cannot commit to a specific timeframe, we expect to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Information on updates will be available in the link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards Knowledgebase article].
+====
+
+[id="new-features-1-30-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.9.
+* {ServerlessProductName} now uses Knative Eventing 1.9.
+* {ServerlessProductName} now uses Kourier 1.9.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.9.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.9.
+* The `kn func` CLI plug-in now uses `func` FIXME.
```

Comment:
Pretty sure you are aware of this, just not to forget :)

Reply:
@ReToCode I am, but I never know the confirmed version :) Usually Lance or Jeff tells me.

Reply:
/cc @lkingland

Reply:
@jrangelramos Jeff, could you tell me the `kn func` version please?